pkg/login/social/social.go

```diff
@@ -113,12 +115,12 @@ func NewOAuthService() {
 		if name == "github" {
 			SocialMap["github"] = &SocialGithub{
 				SocialBase: &SocialBase{
-					Config: &config,
-					log:    logger,
+					Config:         &config,
```

Comment:
Please extract new:ing up new `SocialBase` to a function and use that in all places here and below.